developerportal/templates/molecules/filter-form.html

```diff
@@ -105,6 +105,21 @@ <h5>Country</h5>
     </fieldset>
   {% endif %}
 
+  {% if filters.show_search_input %}
+    <fieldset class="filter-form-section">
+      <header class="filter-form-section-header">
+        <h5>Search</h5>
+          <a href="#" class="filter-form-clear js-filter-clear" data-controls="{{SEARCH_QUERYSTRING_KEY}}">
+          <span class="icon">
+            {% include "atoms/icons/close.svg" %}
+          </span>
+          Clear
+        </a>
+      </header>
+      <input class="filter-form-search js-search-input" name="search" value="" placeholder="Refine with keywords"/>
```

Comment:
This is missing an associated `label` element. Not sure if you have the utility class `visually-hidden`, but that can be used on the label to hide it from everyone except screen reader users.

```
.visually-hidden {
    border: 0;
    clip: rect(0 0 0 0);
    height: 1px;
    margin: -1px;
    overflow: hidden;
    padding: 0;
    position: absolute !important; /* stylelint-disable-line declaration-no-important */
    width: 1px;
}
```